lib/jormungandr/src/Cardano/Wallet/Jormungandr/Compatibility.hs

```diff
@@ -22,6 +22,8 @@ module Cardano.Wallet.Jormungandr.Compatibility
 
 import Prelude
 
+import Cardano.Crypto.Wallet
+    ( XPub (xpubPublicKey) )
```

Comment:
Side-note, one can also import record fields by using their name only:

```hs
import Cardano.Crypto.Wallet
    ( xpubPublicKey )
```

I am fine with both, just find the latter usually more convenient so pointing it out.